### PR TITLE
[Model Monitoring] Encode feature statistics data (KV)

### PR DESCRIPTION
**Pull request by Eyal-Danieli**
When storing a string attribute data under the model endpoint with a huge length, the user might face a v3io-kv string attribute length limitation. 
In this PR we fix that issue by converting the statistics data string (which used to be stored as an attribute of type string) into binary string.

A fix to [ML-4597](https://jira.iguazeng.com/browse/ML-4597).

### CODE REVIEW

File: mlrun/model_monitoring/stores/kv_model_endpoint_store.py [hunk 1]
```diff
@@ -512,3 +535,15 @@ def validate_old_schema_fields(endpoint: dict):
             ] = endpoint[
                 mlrun.common.schemas.model_monitoring.EventFieldType.ENDPOINT_ID
             ]
+
+    @staticmethod
+    def _encode_decode_field(
+        field: typing.Union[str, bytes], encode: bool = True
```
Comment (assaf758) [inline mlrun/model_monitoring/stores/kv_model_endpoint_store.py:541]: No reason to provide default to `encode` param (they are symetric).
Also why provide _endode_field() and _decode_field(), what do we gain in provding both in one function?

**Comment by assaf758** [comment]
Please add an RCA in the jira (a note that it's not a "memory limit error", it's v3io-kv string attribute length limitation)

**Comment by Eyal-Danieli** [comment]
> Please add an RCA in the jira (and note that it's not a "memory limit error", it's v3io-kv string attribute length limitation)

Alex already wrote the RCA in the ticket :)

**Comment by assaf758** [comment]
> > Please add an RCA in the jira (and note that it's not a "memory limit error", it's v3io-kv string attribute length limitation)
> 
> Alex already wrote the RCA in the ticket :)

Not quite, Alex explained what the v3io-engine error-log was about, but it was not clear what is causing this error to appear.
Your explanation here (that the features stats / current stats are stored as one big json in the model-endpoint) completes the picture.

**Review by assaf758 — APPROVED** [review]

**Pull request merged**